website/docs/r/iam_trusted_profiles_claim_rule.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_claim_rule"
+description: |-
+  Manages iam_trusted_profile_claim_rule.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_claim_rule
+
+Create, update, or delete an IAM trusted profiles claim rule resource. For more information, about IAM trusted profiles claim rule, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule
+
+## Example usage
+
+```terraform
+resource "ibm_iam_trusted_profile_claim_rule" "iam_trusted_profile_claim_rule" {
+  conditions = { "claim" : "claim", "operator" : "operator", "value" : "value" }
+  profile_id = "profile_id"
+  type = "type"
+}
+```
+
+## Argument reference
+
+Review the argument reference that you can specify for your resource.
+
+* `conditions` - (Required, List) The conditions of this claim rule.
+Nested scheme for **conditions**:
+	* `claim` - (Required, String) The claim to evaluate against.
+	* `operator` - (Required, String) The operation to perform on the claim. Supported values are EQUALS, NOT_EQUALS, EQUALS_IGNORE_CASE, NOT_EQUALS_IGNORE_CASE, CONTAINS, IN.
+	* `value` - (Required, String) The stringified JSON value that the claim is compared to using the operator.
+* `cr_type` - (Optional, String) The compute resource type the rule applies to, required only if type is specified as 'Profile-CR'. Supported values are VSI, IKS_SA, ROKS_SA.
```

Comment:
These values VSI, IKS_SA, ROKS_SA. must be in bold text as

**VSI**, **IKS_SA**, **ROKS_SA**.